Assignment6Test

```diff
@@ -0,0 +1,119 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+import assignment6.Assignment6;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Stack;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import assignment6.*;
+import java.util.Arrays;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author maria
+ */
+public class Assignment6Test {
+    
+    public Assignment6Test() {
+    }
+     @Test
+   public void noChangeTest() {
+        ArrayList<Character> firstPosition = new ArrayList<>();
+        firstPosition.add('a');
+        firstPosition.add('b');
+        firstPosition.add('c');
+        ArrayList<Character> endPositon = new ArrayList<>();
+        endPositon.add('a');
+        endPositon.add('b');
+        endPositon.add('c');
+        int n = 4;
+        Assignment6 x = new Assignment6(firstPosition,endPositon,n);
+        List<Translation> sol = x.arangeTheCars();
+        List<Translation> empty = new ArrayList<>();
+        assertEquals(sol,empty);
+    }    
+    @Test
+   public void reverseTest() {
+        ArrayList<Character> firstPosition = new ArrayList<>();
+        firstPosition.add('a');
+        firstPosition.add('b');
+        firstPosition.add('c');
+        ArrayList<Character> endPositon = new ArrayList<>();
+        endPositon.add('c');
+        endPositon.add('b');
+        endPositon.add('a');
+        int n = 4;
+        Assignment6 x = new Assignment6(firstPosition,endPositon,n);
+        List<Translation> sol = x.arangeTheCars();
+        String verify = sol.get(0).toString();
```

Comment:
It's clever to use the toString method to verify your solution. An alternative would be to overwrite the equals Method, so you wouldn't need an extra String ;)

Reply:
i do not understand. i have never done it. can you explain me  ?

Reply:
Every class in Java has is equals method. For example if you compare two strings you do it by string1.equals(string2). By overwriting this method for your class Translation you can decide when two objects of your class are equal. Hope that makes it more clear 😉